ledger/participant-state/kvutils/app/src/main/resources/logback.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should reuse the same `logback.xml` across the board instead of adding a new file for every new module.

Reply:
I'm discussing whether this would make sense. It doesn't sound like a bad idea, but I'm a bit worried that coming up with a `logback.xml` that makes everybody happy could not lead to an improvement. In any case, I would not make this part of this PR.

Reply:
Agreed that such change shouldn't be part of this PR. One option is to create a `logback-base.xml` configuration and people can include it and add their favourite classes with favourite log-levels per application:
```xml
<configuration>
  <include resource="logback-base.xml"/>
  
  <logger name="my.favourite.app" level="WARNING"/>
  ...
</configuration>
```

Reply:
Agreed, but not in this PR.